faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reporting Issues and Feature Requests
+area: analysis
+box_type: tip
+layout: faq
+contributors: [jennaj, garimavs]
+---
+
+- For reporting **Usage Problems**, related to tools and functions, head to the [Galaxy Help](https://help.galaxyproject.org/) site.
+    - **Red error datasets**:
+        - To resolve it you might be asked to send a bug report.
+        - Click on the green bug icon within one of the red error datasets to start the submission process 
+    - **Green unexpected-result datasets**:
+        - To resolve it you may be asked to send in a shared history link and possibly a shared workflow link. 
+        - Email our team directly on the private mailing list galaxy-bugs@lists.galaxyproject.org. Include the share link(s) and note the problem dataset numbers involved.
+    - **Functionality problems**:
+        - Using [Galaxy Biostars](https://galaxyproject.org/support/biostar/) is the best way to get help in most cases.
+        - If the problem is more complex, email a description of the problem and how to reproduce it. 
+    - **Administrative problems**:
+        - If the problem is present in your own Galaxy, the administrative configuration may factor.  
+        - For the fastest help directly from the development community, admin issues can be alternatively reported to the mailing list or the GalaxyProject [Gitter channel](https://gitter.im/galaxyproject/Lobby). 
+
+- For **Security Issues**, do not report them via GitHub. Kindly disclose these as explained in this [document](https://github.com/galaxyproject/galaxy/blob/dev/SECURITY_POLICY.md).
+- For **Bug Reporting**, create a Github issue. Include the steps mentioned [here](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting_reporting.html).
+- Search the [Galaxy Search](https://galaxyproject.org/search/) to find prior Q and A, FAQs, Tutorials, and other documentation across all Galaxy resources, to verify in case your issue was already faced by someone.
```

Comment:
If we move all FAQs to GTN this is probably not very relevant. I would recommend to use the GTN search directly for searching.

Reply:
Okay, I'll surely change this part too.